From 7cba2cdc97c910c28636524e2bc7aaf0db1ce9ad Mon Sep 17 00:00:00 2001
From: Erickson Silva de Oliveira <Erickson.SilvadeOliveira@windriver.com>
Date: Mon, 13 Apr 2026 13:49:57 -0300
Subject: [PATCH] ceph-csi-cephfs: replace appVersion/version

This patch changes the appVersion of the cephfs chart to the
upstream version that ceph-csi is being updated to.

Signed-off-by: Erickson Silva de Oliveira <Erickson.SilvadeOliveira@windriver.com>
---
 charts/ceph-csi-cephfs/Chart.yaml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/charts/ceph-csi-cephfs/Chart.yaml b/charts/ceph-csi-cephfs/Chart.yaml
index bc8dd7e..4d3c12d 100644
--- a/charts/ceph-csi-cephfs/Chart.yaml
+++ b/charts/ceph-csi-cephfs/Chart.yaml
@@ -1,10 +1,10 @@
 ---
 apiVersion: v1
-appVersion: canary
+appVersion: 3.16.2
 description: "Container Storage Interface (CSI) driver,
 provisioner, snapshotter, resizer and attacher for Ceph cephfs"
 name: ceph-csi-cephfs
-version: 3-canary
+version: 3.16.2
 keywords:
   - ceph
   - cephfs
--
2.34.1
